client/status/common.go

```diff
@@ -2,10 +2,30 @@ package apistatus
 
 import (
 	"encoding/binary"
+	"errors"
 
 	"github.com/nspcc-dev/neofs-api-go/v2/status"
 )
 
+// ErrAPI describes common error which is a grouping type for any [apistatus] errors. Any [apistatus] error may be checked
+// explicitly via it's type of just check the group via errors.Is(err, [apistatus.ErrAPI]).
+var ErrAPI = errors.New("api error")
+
+var (
+	// ErrServerInternal is an instance of ServerInternal error status. It's expected to be used for [errors.Is]
+	// and MUST NOT be changed.
+	ErrServerInternal ServerInternal
```

Comment:
You can `errors.As` and use its methods. Although practically we don't have a lot of difference there.

Reply:
since that PR exists and is almost merged, i would think if we can keep `ErrServerInternal` only (exported i mean). cause my memory does not answer how `ServerInternal` can be used by an average user if it can not be returned by any method (explicitly, not as `error`)

Reply:
I consider, this `ServerInternal` like 500 in HTTP, you just understand that happened something unexpected